force-app/main/default/classes/QueueablePokemonJob.cls

```diff
@@ -0,0 +1,152 @@
+public class QueueablePokemonJob implements Queueable, Database.AllowsCallouts {
+  public static Integer batchSize = 5;
+  private List<PokeAPICallout.PokemonData> pokemonDataList;
+  public static final String POKE_URL = 'https://pokeapi.co/api/v2/pokemon?';
+  private Integer batchNum;
+  private Integer pokemonLeft;
+  private static Integer numPokemon;
+  private Integer batch;
+  private static Id accountId;
+  private List<Pokemon__c> pokemonList = new List<Pokemon__c>();
+
```

Comment:
Would be nice to see an easier-to-use constructor for when we're initializing this background job for the first time. 

```java
public QueuablePokemonJob(List<PokeAPICallout.PokemonData> pokemonDataList) {
  this.pokemonDataList = pokemonDataList;
  this.pokemonLeft = 0;
  this.batchNum = 0;
  this.batch = 0;
}
```